tonyneel923/front-end-bootstrap
# Feature/form validation

To Test:

Navigate to the register user page.

try hitting submit leaving everything blank to validate error messages. 

take a look at the coded requirements for various fields and verify error messages display when they are not met. 

Verify success messages appear when entered data meets requirements.
